File: src/utils/functions/ePower.js
import {factorial} from './factorial'

export function ePower (num) {
  let retained = 1
  let nominator
  let denominator
  let rounds = 40

  for (let i = 1; i <= rounds; i++) {
    nominator = Math.pow(num, i)
    denominator = factorial(i)
    retained = retained + nominator / denominator
  }

  return retained
}

File: app/functions/ePower.py
from .factorial import factorial

def ePower (num):
  from .power import power
  retained = 1
  nominator = 0
  denominator = 0
  rounds = 40

  for i in range(1,rounds+1):
    nominator = power(num, i)
    denominator = factorial(i)
    retained = retained + nominator / denominator
    
  return retained
